roles/kube_vip/defaults/main.yml

```diff
@@ -20,3 +20,18 @@ kube_vip_interface: "{{ keepalived_interface | default(kubernetes_keepalived_int
 
 # IP address to use for kube-vip
 kube_vip_address: "{{ keepalived_vip | default(kubernetes_keepalived_vip) }}"
+
+# use bgp for kube-vip
+kube_vip_bgp_enabled: false
+
+# BGP peers, if BGP is enabled
+kube_vip_bgp_peers: 192.168.0.10:65000::false,192.168.0.11:65000::false
```

Comment:
Same here!

Reply:
made change , left kube_vip_mode: arp as default so it doesnt break anyone existing.